peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -5,44 +5,46 @@
 
 {% block page-content %}
   <main class="pb-5">
-    <div class="container">
-      {% block breadcrumbs %}
-      {% endblock %}
+    <div class="bg-light d-flex">
```

Comment:
Why d-flex?

Reply:
![image](https://user-images.githubusercontent.com/15995210/193225735-27d3a24b-50d7-45be-969b-0175b031c5fc.png)
This was an escaping margin-top, that made a gap between the main and body elements. Super strange.
`d-flex` worked to fix this issue, but actually using `py-4` on `h1` is a clearer solution.

Reply:
It seems to be because of collapsing margins 🤔
https://stackoverflow.com/questions/17137262/why-does-the-child-divs-margin-affect-the-margin-of-the-parent